doc/syslog/Syslog_Source_IP_test_plan.md

```diff
@@ -0,0 +1,225 @@
+# Syslog Source IP Test Plan
+
+## Related documents
+
+| **Document Name** | **Link** |
+|-------------------|----------|
+| Syslog Source IP HLD | [https://github.com/sonic-net/SONiC/blob/983fb02b909c236e06f2849ee990d4f3fd377516/doc/syslog/syslog-design.md]|
```

Comment:
@JibinBao please either update with `https://github.com/sonic-net/SONiC/blob/master/doc/syslog/syslog-design.md` or remove

Reply:
Update is done